src/main/webapp/WEB-INF/tags/instructor/search/searchPageInput.tag

```diff
@@ -32,20 +32,20 @@
                 class="glyphicon glyphicon-info-sign">
             </span>
           </li>
+          <li>
+            <input id="students-check" type="checkbox"
+                   name="<%=Const.ParamsNames.SEARCH_STUDENTS%>" value="true"
+                   <c:if test="${data.searchForStudents || !data.searchCommentForResponses}">checked</c:if>>
+            <label for="students-check">
+              Students
+            </label>
+          </li>
           <li>
             <input id="comments-for-responses-check" type="checkbox"
```

Comment:
Let's change the `id` to what the input is for. For e.g. this `question-responses-comments-on-responses` ?

Reply:
Maybe something more general and shorter like `session-feedback` would be good for it?

Reply:
That's too general. I think `search-questions-responses-comments-check` is good.

Reply:
On second thoughts, `search-session-feedback-data-check` is good.

Reply:
Updated to proposed value